app/services/common_platform/api/prosecution_case_fetcher.rb

```diff
@@ -2,7 +2,7 @@
 
 module CommonPlatform
   module Api
-    class ProsecutionCaseSearcher < ApplicationService
+    class ProsecutionCaseFetcher < ApplicationService
```

Comment:
Renamed to avoid confusion with `SearchProsecutionCase` and to adopt an existing naming convention.